specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/security.json

```diff
@@ -0,0 +1,278 @@
+            "description": "Error response structure.",
+            "properties": {
+                "error": {
+                    "x-ms-client-flatten": true,
+                    "description": "Error data",
+                    "$ref": "#/definitions/CloudErrorBody"
+                }
+            }
+        },
+        "CloudErrorBody": {
+            "x-ms-external": true,
+            "type": "object",
+            "description": "Error details.",
+            "properties": {
+                "code": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+                },
+                "message": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "A message describing the error, intended to be suitable for display in a user interface."
+                }
+            }
+        },
+        "Resource": {
+            "type": "object",
+            "description": "Describes an Azure resource.",
+            "properties": {
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource Id"
+                },
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name"
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource type"
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "PricingList": {
+            "description": "List of pricing configurations response",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of pricing configurations",
+                    "items": {
+                        "$ref": "#/definitions/Pricing"
+                    }
+                }                
+            }
+        },
+        "Pricing": {
+            "type": "object",
+            "description": "Pricing tier will be applied for the scope based on the resource ID",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Pricing data",
+                    "$ref": "#/definitions/PricingProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "PricingProperties": {
+            "type": "object",
+            "description": "Pricing properties for the relevant scope",
+            "properties": {
+                "pricingTier": {

```

Comment:
Isnt this similar notion as a SKU? If yes, it should be modeled using the sku property which is a top level property. More details can be found in ARM Resource provider contract.

Reply:
It's a new API version to an existing type, customers are using it and used to use it. I don't think we want to deprecate it and use the SKU propery instead as it will be very confusing to existing customers.
I'm also not sure we can use the SKU property even if we wanted to since this pricing type implies the tier of the subscritpion in Microsoft.Secruity, and not the sku of a specific resource.